include/vcpkg/base/messages.h

```diff
@@ -755,14 +764,36 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Both a manifest file and a CONTROL file exist in port directory: {path}");
+    DECLARE_MESSAGE(ControlCharacterInString, (), "", "Control character in string");
     DECLARE_MESSAGE(CopyrightIsDir, (msg::path), "", "`{path}` being a directory is deprecated.");
     DECLARE_MESSAGE(CorruptedDatabase, (), "", "Database corrupted.");
     DECLARE_MESSAGE(CorruptedInstallTree, (), "", "Your vcpkg 'installed' tree is corrupted.");
     DECLARE_MESSAGE(CouldNotDeduceNugetIdAndVersion,
                     (msg::path),
                     "",
                     "Could not deduce nuget id and version from filename: {path}");
+    DECLARE_MESSAGE(CouldNotFindBaseline,
+                    (msg::value, msg::path),
+                    "{value} is a commit sha",
+                    "Could not find explicitly specified baseline `\"{value}\"` in baseline file {path}");
+    DECLARE_MESSAGE(CouldNotFindBaselineForRepo,
+                    (msg::value, msg::package_name),
+                    "{value} is a commit sha.",
+                    "Couldn't find baseline `\"{value}\"` for repo {package_name}");
+    DECLARE_MESSAGE(FailedToFetchError,
+                    (msg::error_msg, msg::package_name, msg::error),
+                    "{error} is the error code",
+                    "{error_msg}\nFailed to fetch {package_name}:\n{error}");
+    DECLARE_MESSAGE(CouldNotFindBaselineInCommit,
+                    (msg::value, msg::package_name, msg::error_msg),
+                    "{value} is a commit sha.",
+                    "Couldn't find baseline in commit `\"{value}\"` from repo {package_name}:\n{error_msg}");
+    DECLARE_MESSAGE(CouldNotFindGitTreeAtCommit,
+                    (msg::package_name, msg::value, msg::error_msg),
+                    "{value} is a commit sha",
+                    "could not find the git tree for `versions` in repo {package_name} at commit {value}: {error_msg}");
```

Comment:
I think these error_msgs should probably be printed on their own rather than being included in the localized part.